Libraries/Drei/base.lisp

```diff
@@ -786,7 +786,7 @@ in the region delimited by mark1 and mark2."))
                                               offset
                                               tab-width))
                (count (- tab-width (mod column tab-width))))
-          (delete-buffer-range buffer offset 1)
+          (delete-buffer-range buffer offset count)
           (loop repeat count
```

Comment:
if this change is necessary please make it a separate commit and explain why it is necessary.

Reply:
please have a look into the issue #678 (probably  #672 too) , there I already described it.